portal/views/patients.py

```diff
@@ -46,23 +51,33 @@ def patients_root():
         ).join(UserOrganization).filter(
             and_(UserOrganization.user_id==User.id,
                  UserOrganization.organization_id.in_(org_list)))
+    leaf_organizations = None
+    try:         
+        leaf_organizations = user.leaf_organizations()
+    except ValueError:
+         current_app.logger.debug("In patients list: error retrieving leaf organizations for user %s", str(user.id))
```

Comment:
Paul, the ValueError is raised for a provider who doesn't have an org affiliation yet (or "None of the above" is selected" for this provider)